bigchaindb_driver/driver.py

```diff
@@ -194,22 +194,38 @@ def fulfill(transaction, private_keys):
         return fulfill_transaction(transaction, private_keys=private_keys)
 
     def get(self, *, asset_id, operation=None, headers=None):
-        """Get a list of transactions that have the given asset id and or
-        operation.
+        """Given an asset id, get its list of transactions (and
+        optionally filter by the operation type (``'CREATE'`` or
```

Comment:
I might rephrase the clause inside the brackets to something like

> (and optionally filter for only ``'CREATE'`` or ``'TRANSFER'`` transactions)

to avoid the nested brackets (not really a thing ~~(really)~~).